src/test/resources/storage-loan-6-4.json

```diff
@@ -0,0 +1,108 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
Usually I’d expect this to be a move rather than a new file created in the diff. Has the old schema been removed?

Reply:
No, the old schema is still there. I'll move 6.2 to 6.4

Reply:
Removed.